docs/content/users/usage/commands.md

```diff
@@ -227,6 +227,16 @@ Run the `cake` command; available only in projects of type `cakephp`, and only a
 ddev cake
 ```
 
+## `cd`
+
+Uses shell built-in `cd` to change to a project directory.
+
+```shell
+# Goes to the project-name directory
+# Works only if you applied the instructions from this command
+ddev cd project-name
+```
```

Comment:
```suggestion
Uses shell built-in `cd` to change to a project directory. For example, `ddev cd some-project` will change directories to the project root of the project named `some-project`. 

Note that this command can't work until you make a small addition to your `.bashrc`, `.zshrc`, or `config.fish`. To see the explanation of what you need to do:

`ddev cd show-me-how`
```

Reply:
I updated `ddev cd show-me-how` part - because it showed this error:

```
$ ddev cd show-me-how
Failed to find path for project: could not find a project named 'show-me-how'. Run 'ddev list' to see currently active projects
```

The real name of the project should be provided here to show the help.